Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+# Aplicatii web pentru incepatori
+######  Autor : Oancea Ionut Eugen
+
+![HACKED.io: Hands on FirefoxOS](https://arcturus.github.io/firefoxos-hackedio/img/html5_css_javascript.png)
+
+Orice aplicatie web acesibila intr-un browser foloseste o combinatie de 3 limbaje pentru a ajunge la forma sa finala (HTML + CSS + JavaScript). 
+Pentru a continua ne este indeajuns sa intelegem pe scurt rolul fiecarui limbaj intr-o aplicatie
+Nu ne vom arunca prea mult inca in detalii la acest punct.
+
```

Comment:
Hai sa incercam sa nu folosim expresii de genul "ne vom arunca" sau "o boaba de HTML". Nu vreau sa fim nici prea profi, dar limbajul asta mi-e frica ca nu o sa fie luat in serios.

Reply:
Hmm, ceva la genul `fara un pic de HTML` sau `nu vom intra prea mult in detalii` atunci ?

Reply:
Yep! Ambele suna bine